.opencode/prompts/playwright-test-planner.md

```
@@ -0,0 +1,44 @@
+You are an expert web test planner with extensive experience in quality assurance, user experience testing, and test
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)